B/blis/build_tarballs.jl

```diff
@@ -0,0 +1,91 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "blis"
+version = v"0.7.0+0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/flame/blis.git", "68b88aca6692c75a9f686187e6c4a4e196ae60a9")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd blis/
+
+for i in ./config/*/*.mk; do
+
+    # Building in container forbids -march options
+    sed -i "s/-march[^ ]*//g" $i
+    # Building in container forbids unsafe optimization.
+    sed -i "s/-ffast-math//g" $i
+    sed -i "s/-funsafe-math-optimizations//g" $i
+
+done
+
+case ${target} in
+
+    *"x86_64"*"linux"*) 
+        export BLI_CONFIG=x86_64
+        export BLI_THREAD=openmp
+        ;;
+    *"x86_64"*"w64"*) 
+        # MinGW doesn't support savexmm instructions
+        # Build only for AMD processors.
+        export BLI_CONFIG=amd64
+        export BLI_THREAD=openmp
+        ;;
+    *"x86_64"*"apple"*) 
+        export BLI_CONFIG=x86_64
+        export BLI_THREAD=none
```

Comment:
You can use OpenMP for macOS and FreeBSD, if you want, but you need to use GCC as compiler: https://juliapackaging.github.io/BinaryBuilder.jl/latest/build_tips/#Using-GCC-on-macOS-and-FreeBSD

Reply:
Sorry I'm not familiar with the build environment.
So in fact `openblas.dylib` binary shipped in julialang.org is also links to this gomp, I guess?

Thanks for the information!

Reply:
I'm not entirely sure, bit I think OpenBLAS doesn't use openmp :thinking:

Reply:
Off-topic but I'm able to change number of threads with `OMP_NUM_THREADS` when doing `Array*Array`. :smile: